src/helpers/tests/geoChart/fixtures.ts

```diff
@@ -75,3 +75,77 @@ export const COLOR_ITEM: VisualizationObject.IBucket = {
         },
     ],
 };
+
+export const LOCATION_ITEM_BY_IDENTIFIER: VisualizationObject.IBucket = {
```

Comment:
It should be `LOCATION_BUCKET_WITH_ITEM_BY_IDENTIFIER `
Please fix others

Reply:
fixed